src/lplib/option.ml

```diff
@@ -38,3 +38,9 @@ let eq : 'a eq -> 'a option eq = fun eq_elt o1 o2 ->
 
 let pp : 'a pp -> 'a option pp = fun pp_elt ppf o ->
   match o with None -> () | Some e -> pp_elt ppf e
+
+module Binding = struct
+  let ( let* ) : 'a t -> ('a -> 'b t) -> 'b t = fun o f -> 
+    match o with None -> None | Some x -> f x
+  let return x = Some x
+end
```

Comment:
let* is bind with arguments permuted. It would be better to replace bind by let* and do not put let* and return in a module.

Reply:
I propose to leave the `bind` for now: I agree it would be better to replace bind by `let*` (it's more ocamlish IMO), but I'd like it to be done in an other PR.

Reply:
Ok but if you finally have a total translation function for types, there is no need for this new module.

Reply:
Furthermore, indeed.